### PR TITLE
KCI-141 Always redirect user to loginpage with success-message

### PR DESCRIPTION
**Pull request by martn001**

### CODE REVIEW

File: generator/templates/Authorisation/src/views/authorisation/RegistrationVerify.vue [hunk 1]
```diff
@@ -86,8 +86,7 @@ export default {
 
       verify(token, this.form.email, this.form.password, this.form.passwordConfirmation)
           .then(() => {
-            this.alertMessage = this.$t('authorisation.registrationVerify.successMessage');
-            this.alertType = 'success';
+            this.$router.push({name: 'login', query: {message: 'registrationVerify'}});
```
Comment (martn001) [inline generator/templates/Authorisation/src/views/authorisation/RegistrationVerify.vue:89]: I thought there also should be a redirection after when registration has been verified.

**Review by jvhellemondt — APPROVED** [review]
Akkoord zolang de gebruiker wel duidelijk kan zien dat zijn registratie gelukt is.

File: generator/templates/Authorisation/src/locales/nl/authorisation.json [hunk 2]
```diff
@@ -1,7 +1,12 @@
 {
     "login": {
         "title": "Inloggen",
-        "successMessage": "Je bent ingelogd, we sturen je nu door."
+        "successMessage": "Je bent ingelogd, we sturen je nu door.",
+        "messages": {
+            "invitationAcceptSuccess": "Je account is geactiveerd, je kunt nu inloggen.",
+            "passwordResetSuccess": "Je wachtwoord is veranderd, je kunt nu inloggen.",
+            "registrationVerify": "Je registratie is voltooid, je kunt nu inloggen."
+        }
```
Comment (ricardovanlaarhoven) [inline generator/templates/Authorisation/src/locales/nl/authorisation.json:9]: You forgot to remove these messages below (i quess where you copied them from?)
for example line 23

**Review by ricardovanlaarhoven — CHANGES_REQUESTED** [review]

File: generator/templates/Authorisation/src/locales/nl/authorisation.json (hunk 2 above)
Comment (ricardovanlaarhoven) [reply generator/templates/Authorisation/src/locales/nl/authorisation.json:9]: Really weird, when opening this branch local they were removed..

**Pull request merged**